### DISCUSSION TITLE
在线音乐播放器该如何配置，谢谢

### DISCUSSION THREAD

**Discussion by always1314** [mirai-mamori/Sakurairo#364] [Q&A]
> 注意，在报告之前请先确认：**这个想法能被多数人接受；这个想法会让这个主题更好。**
> 当你确定你的想法满足以上条件之后，请删除这段文本并继续。

**这是什么方面的想法？**

现有内容的优化 / 新增功能 / UI 界面 / 用户体验

**这个想法具体是要实现什么？**

**这个想法实现后有什么帮助？**

**是否有具体的实现思路？**

**补充信息：**

**Comment by KotoriK** [comment]
求助请发Discussion 谢谢

**Reply by always1314** [reply]
OK

**Reply by always1314** [reply]
就页脚的版权被在线播放音乐播放器挡住了，怎么把页脚下面的信息往上面移动

**Reply by KotoriK** [reply]
确认是个bug 将在下版本修复 感谢你的支持
